src/main/java/edu/wpi/first/pathui/FxUtils.java

```diff
@@ -22,4 +25,37 @@ public static MenuItem menuItem(String text, EventHandler<ActionEvent> eventHand
     return menuItem;
   }
 
+  /**
+   * Get the color according to sub-child order.
+   * @param i The index of the child relative to the parent
+   * @return The color to draw the thing as
+   */
+  public static Color getColorForSubChild(int i) {
+    Color[] colors = new Color[]{Color.ORANGE, Color.VIOLET, Color.DARKRED};
+    return colors[i % colors.length];
+  }
+
+  /**
+   * Check whether the item is a child of a child (exactly the child of a child of the root.)
+   * @param view The treeview to check against
+   * @param item The item to check
+   * @return Whether this is a sub child
+   */
+  public static boolean isSubChild(TreeView view, TreeItem item) {
+    return view.getTreeItemLevel(item) == 2;
```

Comment:
Is there a better way to check this?

Reply:
That is probably my fault. TreeViews requires that every treeItem have the same object type which seemed cumbersome. Need to hold

Strings (The top level, text "Paths" and "Autons")
Paths (some object representing a sub child of an auton, preferably without going through the effort actually parsing the path files that are  not needed yet)
Auton (some object representing a first level child in the "Autons" treeView)

It is currently implemented where everything is just a string, either the title of the section or the filename. There has been some discussion of walking back the initial decision so easy helper functionality like
if(item instanceOf PathItem) 
could be available. Currently the only way to get what type of item is by its position in the TreeView. 
I have used
item.getParent().getParent() == null for similar checks.

Reply:
There's a lot of non-ideal coding in this that's just the result of TreeView overall not being quite built for this situation. This kind of check wouldn't be necessary if it were possible to restyle TreeItems without changing the TreeCell, for example. I'd love to have a better way, but after a good bit of searching last night this is the most elegant solution I could come up with.